src/h5web/guards.ts

```diff
@@ -25,6 +26,10 @@ export function isDefined<T>(val: T): val is T extends undefined ? never : T {
   return val !== undefined;
 }
 
+export function isReactElement(child: ReactChild): child is ReactElement {
```

Comment:
It could be clearer to use the real type rather than the alias (i.e. `ReactElement | string | number` instead of `ReactChild`) but that's all right like this

Reply:
yeah, I might do this in the last PR then, to avoid having to rebase 👍